04-NodeJS-NPM-Modules/modules/Movie.js

```diff
@@ -0,0 +1,22 @@
+function Movie(title, genre, minutes, rating) {
+    this.title = title;
+    this.genre = genre;
+    this.minutes = minutes;
+    this.rating = rating;
+    this.director;
+
+    console.log("Se ha creado una instancia del objeto movie");
+}
+
+require('./Director').director;
```

Comment:
require() will return what's been export with module.exports. It doesn't execute anything. Therefore, you should store its returned value into a variable.